### ISSUE TITLE
Microsoft Edge Cleanup Error

### ISSUE THREAD

**Issue by RJHornby01** [bleachbit/bleachbit#1539]
When I run Bleach bit it cleans everything else but when it get to Microsoft Edge it come up with the following error.
![Screenshot (10)](https://github.com/bleachbit/bleachbit/assets/144386395/867c1dba-3ee6-480a-82ec-6039daf3ca00)

Have uninstalled and reinstalled Bleach bit but still the same.

Please advice on how to fix it.

Thank you.

Richard Hornby

**Comment by az0** [comment]
That screenshot nicely explains your problem, so thanks.

Please see the [BleachBit version 4.4.2 release notes for "no such table"](https://www.bleachbit.org/news/bleachbit-442#no_such_table)

**Comment by az0** [comment]
@RJHornby01 
[BleachBit version 4.5.0 beta ](https://www.bleachbit.org/news/bleachbit-450-beta) has this fix.  Please test and let me know how it goes.

**Comment by RJHornby01** [comment]
Have checked the Windows Directory and its empty %WinDir%\System32\config\Systemprofile\AppData\Local\Microsoft\Windows\INetCache\. Did copy a document over to it but when I used BleachBit with Admin Privileges it didn't clear it but have manually cleared it.

Don't have a C:\Windows.old folder. Have used BleachBit to clean Windows Updates.

I took of Microsoft Edge from the BleachBit list to clean, but when I put it on again it still comes up with the same error. Have run it with Admin Privileges and normal. Have also restarted PC to see if that fixes it it but doesn't.

Everything else cleans well.

Have also reset Microsoft Edge in case that was causing it, but sill the same.
